pkg/collector/corechecks/containers/docker/docker.go

```diff
@@ -434,9 +432,30 @@ func (d *DockerCheck) Configure(config, initConfig integration.Data, source stri
 		log.Warnf("Can't get container include/exclude filter, no filtering will be applied: %w", err)
 	}
 
+	d.setOkExitCodes()
+
 	return nil
 }
 
+// setOkExitCodes defines the ok exit codes based on
+// the default values and the OkExitCodes config field.
+func (d *DockerCheck) setOkExitCodes() {
+	d.okExitCodes = map[int]struct{}{}
+
+	// Apply custom ok exit codes if defined.
+	if len(d.instance.OkExitCodes) > 0 {
+		for _, code := range d.instance.OkExitCodes {
+			d.okExitCodes[code] = struct{}{}
+		}
+
+		return
+	}
+
+	// Set default ok exit codes.
+	// 143 is returned when docker sends a SIGTERM to stop a container.
+	d.okExitCodes = map[int]struct{}{0: {}, 143: {}}
```

Comment:
I'm wondering if we should add 143 in the default values.
I think it is better to support "exit code" linux standard, but we are changing a bit the default behaviour.